.github/workflows/deploy.yml

```
@@ -99,14 +99,17 @@ jobs:
           ssh ${{ secrets.USER }}@${{ secrets.HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
           mkdir ${{ env.PAYU_ENVIRONMENT_LOCATION }}
           if [ $? -ne 0 ]; then
-            exit $?
+            # Replace pre-existing environment
+            echo "Removing pre-existing environment at: ${{ env.PAYU_ENVIRONMENT_LOCATION }}"
```

```suggestion
            echo "::notice::Removing pre-existing environment at: ${{ env.PAYU_ENVIRONMENT_LOCATION }}"
```
Just making it more visible since it's quite a big change to remove an existing environment. Debating whether to put it as a warning, even, instead of a notice.

---

Good point, I think `::warning::` would make sense - Added in cc6083b